### ISSUE TITLE
Test-IcingaForWindowsMigration fails if more than one framework-version is found on the system

### ISSUE THREAD

**Issue by K0nne** [Icinga/icinga-powershell-framework#615]
I found a strange behaviour on a test system of mine. `use-icinga` failed with the following error:

```
PS C:\Windows\system32> use-icinga
Cannot convert the "System.Object[]" value of type "System.Object[]" to type "System.Version".
At C:\Program Files\WindowsPowerShell\Modules\icinga-powershell-framework\cache\framework_cache.psm1:3962 char:5
+     [Version]$CurrentFrameworkVersion = (Get-Module -ListAvailable -N ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [], RuntimeException
    + FullyQualifiedErrorId : ConvertToFinalInvalidCastException
```

## Expected Behavior
I think it is just a mistake of mine, but maybe this function should alert, if  concurrent versions are found:

```
PS C:\Windows\system32> (Get-Module -ListAvailable -Name icinga-powershell-framework).Version;

Major  Minor  Build  Revision
-----  -----  -----  --------
1      10     1      -1
1      0      2      -1
1      2      0      -1
```

## Possible Solution
Fail with a expressive error message.

**Comment by LordHepipud** [comment]
Thanks for the issue. We will fix this with v1.11.0. We will apply the migration to the latest version and print a warning, if multiple frameworks are found.

**Comment by LordHepipud** [comment]
```powershell
[Warning]: Found multiple installations of the module "icinga-powershell-framework". Please check the list below and cleanup your installation to ensure system integrity
 => Path "C:\Program Files\WindowsPowerShell\Modules\icinga-powershell-framework" with version "1.10.0"
 => Path "C:\Development\WindowsPowerShell\modules\icinga-powershell-framework" with version "1.11.0"
